test/common/testlib.py

```diff
@@ -2154,31 +2154,57 @@ def get_decorator(method, _class, name, default=None):
 #
 
 def skipBrowser(reason: str, *args: str):
+    """Skip browser decorator
+
+    Skips a test for provided *reason* on *args* browsers.
+    """
+
```

Comment:
No extra newline here, please.

Maybe make the text a bit less redundant? "Decorator for skipping a test on given browsers" - "You need to provide a reason.". And perhaps rename `args` to `browsers`?
    
```